test/rekt/resources/eventtype/eventtype.go

```diff
@@ -64,14 +69,21 @@ func AssertPresent(expectedCeTypes sets.String) EventType {
 	return EventType{
 		Name: "test eventtypes match or not",
 		EventTypes: func(etl eventingv1beta2.EventTypeList) (bool, error) {
```

Comment:
Not sure if this is the best place to tackle it, but while you are fixing the logic in this function it might be worth switching `expectedCeTypes` from type `sets.String` to `Set[string]{}` as `sets.String` is now deprecated

Reply:
New issue created #7167